smart-contracts/contracts/mixins/MixinKeys.sol

```diff
@@ -200,14 +198,13 @@ contract MixinKeys is
   }
 
   /**
-   * Public function which returns the total number of unique owners (both expired
+  * @dev Public function which returns the total number of unique owners (both expired
    * and valid).  This may be larger than outstandingKeys.
```

Comment:
I need to look at this closer, but wouldn't `outstandingKeys` be a closer match for `totalSupply` than `numberOf Owners` (which may be greater than `outstandingKeys`)?

Reply:
I guess `numberOfKeysSold` is what I meant here. But looking closer,
If one key has been purchased on a lock and the owner transfers it to a new owner:
- there are now 2 owners in the ownersAray
- there are technically 2 keys on the lock, both with the same `tokenId`, but one is expired. I feel like we should be treating this as though there's only one key, and in that case `numberOfOwners` would not be equal to `totalSupply`.

Reply:
The fact that we technically have 2 keys in this case is more an artifact of the way we exacute a transfer than a desired outcome. At least that's what my caffeine-deprived brain says at the momment;)

Reply:
Yea you are right, my mistake.  Good catch - will switch to replacing outstandingKeys.

Reply:
K fixed.  Thanks again, good catch.